files/image_config/interfaces/interfaces-config.sh

```diff
@@ -18,6 +18,8 @@ fi
 
 [ -f /var/run/dhclient.eth0.pid ] && kill `cat /var/run/dhclient.eth0.pid` && rm -f /var/run/dhclient.eth0.pid
 
+ip addr flush dev lo
```

Comment:
Interesting, `ifdown `is supposed to flush the ip addresses. Looks like `ifdown `wouldn't have succeeded in this case and may have been fixed by the recent change. Can you check this PR - https://github.com/Azure/sonic-buildimage/pull/4149?

Reply:
@prsunny No, the ifdown only flush the addresses that are in /etc/network/interfaces, but since the configurations in  /etc/network/interfaces are changing, that is exactly why we get more and more ip addresses.

Eg, The ip addresses was changed to 10.10.10.1/32 and  2000::1/128 in configDB, even if ifdown, it will preserve the old ip addresses.

```
admin@sonic:~$ sudo ifdown --force lo;
admin@sonic:~$ ip addr show lo
1: lo: <LOOPBACK> mtu 65536 qdisc noqueue state DOWN group default qlen 1
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet 10.10.10.2/32 brd 10.10.10.2 scope global lo
       valid_lft forever preferred_lft forever
    inet6 2000::2/128 scope global tentative 
       valid_lft forever preferred_lft forever
```